### ISSUE TITLE
the training details for remastering

### ISSUE THREAD

**Issue by qianmingduowan** [satoshiiizuka/siggraphasia2019_remastering#3]
could you provide some more details for training?
I want to know whether training from scratch? or need another step?
In addition, I would like to know the learning rate strategy，which is not explicitly stated in the paper.

**Comment by Ssakura-go** [comment]
> could you provide some more details for training? I want to know whether training from scratch? or need another step? In addition, I would like to know the learning rate strategy，which is not explicitly stated in the paper.

Have you gotten the related information? I'm confused about this question too.

**Comment by qianmingduowan** [comment]
No,  I have change my research interest.

**Comment by qianmingduowan** [comment]
> > could you provide some more details for training? I want to know whether training from scratch? or need another step? In addition, I would like to know the learning rate strategy，which is not explicitly stated in the paper.
> 
> Have you gotten the related information? I'm confused about this question too.


No, I have change my research interest.

**Comment by Ssakura-go** [comment]
> > > could you provide some more details for training? I want to know whether training from scratch? or need another step? In addition, I would like to know the learning rate strategy，which is not explicitly stated in the paper.
> > 
> > 
> > Have you gotten the related information? I'm confused about this question too.
> 
> No, I have change my research interest.

Roger that. Thank you for your quick reply~
